graph/config/cytoscape/cytoscape.go

```diff
@@ -471,6 +473,19 @@ func buildConfig(trafficMap graph.TrafficMap, nodes *[]*NodeWrapper, edges *[]*E
 			*edges = append(*edges, &ew)
 		}
 	}
+
+	// TODO: Just for Ambient
+	for _, e := range *edges {
+		if e.Data.Waypoint == appender.WaypointTo {
+			for _, otherEdge := range *edges {
+				if e.Data.ID != otherEdge.Data.ID && e.Data.Source == otherEdge.Data.Target && e.Data.Target == otherEdge.Data.Source {
+					e.Data.WaypointEdge = otherEdge.Data
+					// Delete the other edge?
+				}
+			}
+		}
+	}
+
```

Comment:
I'm not sure this is the best place for this logic.  In Cytoscape.go we are really just trying to translate the traffic map into the cytoscape json format (we use the same format for PFT, it's just that for Cy this format can be loaded automatically as a blob, and for PFT we have to parse it out ourselves).

Should this maybe be done in the ambient appender?

Is it necessary to have two separate Waypoint fields in the EdgeData?  Should we combine `EdgeData.Waypoint` and `EdgeData.WaypointEdge` into a single structure?  Something like:

```
type WaypointEdgeData struct {
	Direction string // to | from
	FromEdge *EdgeData // for a 'to' waypoint edge, this is the return 'from' edge
}

EdgeData.Waypoint: WaypointEdgeData

Reply:
I have added this structure in the last commit. 

I will review about adding the logic in the Ambient appender. I agree is not the best place, but I've found some issues for adding it in the appender (I think it was related to that in the beginning I was using the ID to just link the edge, and it was not available until this point, but now that the node is attached, it might be possible).

Reply:
Moved this logic to the Appender, there was the need to redo the Cytoscape edge conversion.